Umee-Bots/lendingPool-reentrancy/README.md

```diff
@@ -0,0 +1,18 @@
+# Detect Reentrancy In Lending Pool
+
+## Description
+
+Detects reentrancy in any call to LendingProtocol. If that’s the case, emits an alert.
```

Comment:
This was changed, so it's better to make it clear in the README. Now we only emit an alert if the reentrant call function is in a blacklist.